src/authenticatedFetch/unauthenticated/UnauthenticatedFetcher.ts

```diff
@@ -30,6 +30,7 @@ import { IDpopHeaderCreator } from "../../dpop/DpopHeaderCreator";
 import { IUrlRepresentationConverter } from "../../util/UrlRepresenationConverter";
 import { IStorageUtility } from "../../localStorage/StorageUtility";
 import { flattenHeaders } from "../headers/HeadersUtils";
+import { Headers as NodeHeaders } from "node-fetch";
```

Comment:
Not used?

Reply:
Resolved in d084878001cd35bdca8cc97b48d46423fad32c1a.